src/security.py

```
@@ -94,12 +94,11 @@ async def verify_jwt(token: str) -> JWTParams:
             params.exp = decoded["exp"]
-            if (
-                params.exp
-                and datetime.datetime.fromisoformat(params.exp)
-                < datetime.datetime.now()
-            ):
-                raise AuthenticationException("JWT expired")
+            if params.exp:
+                exp_time = parse_datetime_iso(params.exp)
+                current_time = datetime.datetime.now(datetime.timezone.utc)
+                if exp_time < current_time:
+                    raise AuthenticationException("JWT expired")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)